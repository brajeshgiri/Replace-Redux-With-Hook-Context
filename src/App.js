import React, { Component } from 'react';
import logo from './logo.svg';
import './App.css';

class App extends Component {
  state = { envName: '' }
  componentDidMount() {
    const envName = process.env.envName;
    this.setState({ envName });
  }

  render() {
    return (
      <div className="App">
        <div className="App-header">
          <img src={logo} className="App-logo" alt="logo" />
          <h2>Hello Vandana</h2>
        </div>
        <p className="App-intro">
          Welcome this is your website live
        </p>
        <p>{this.state.envName}</p>
      </div>
    );
  }
}

export default App;
