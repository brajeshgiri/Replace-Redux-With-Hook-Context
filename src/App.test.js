import React from 'react';
import ReactDOM from 'react-dom';
import App from './App';
import { create } from "react-test-renderer";

describe('App Component', () => {

  it("Matches the snapshot", () => {
    const app = create(<App />);
    expect(app.toJSON()).toMatchSnapshot();
  });
});

