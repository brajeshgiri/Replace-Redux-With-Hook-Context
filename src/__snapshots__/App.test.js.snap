exports[`App Component Matches the snapshot 1`] = `
<div
  className="App">
  <div
    className="App-header">
    <img
      alt="logo"
      className="App-logo"
      src="logo.svg" />
    <h2>
      Hello Vandana
    </h2>
  </div>
  <p
    className="App-intro">
    Welcome this is your website live
  </p>
  <p>
    Env: 
  </p>
</div>
`;
